libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -25,12 +25,25 @@
 #include <inttypes.h>
 
 #include <AP_Math/AP_Math.h>
+#include "AP_Math/fourier.h"
 
 #include "AP_InertialSensor.h"
 
+
+const int SAMPLE_RATE=1600;
+const int MIN_FREQ=1;
+const int BUF_SIZE=1600;//SAMPLE_RATE/MIN_FREQ;
+
 class AuxiliaryBus;
 class DataFlash_Class;
 
+struct log_data{
```

Comment:
you created struct log_data but never use it. If you do use it, it would make sense to use a Vector3f Acc

Reply:
erased